bookdown/04-pipelines.Rmd

```diff
@@ -427,40 +436,40 @@ We first create a "level 0" learner, which is used to extract a lower level pred
 Additionally, we `clone()` the learner object to obtain a copy of the learner.
 Subsequently, one sets a custom id for the `r ref("PipeOp")` .
 
-```{r 04-pipelines-038}
+```{r 04-pipelines-039, eval = FALSE}
 lrn = mlr_learners$get("classif.rpart")
 lrn_0 = PipeOpLearnerCV$new(lrn$clone())
 lrn_0$id = "rpart_cv"
 ```
 
-We use `r ref("PipeOpNOP")`  in combination with `r ref("gunion")`, in order to send the unchanged Task to the next level.
-There it is combined with the predictions from our decision tree learner.
+We use a 'FIXME' in parallel to the "level 0" learner.
```

Comment:
This must be a merging mistake, right? Why would you overwrite `r ref("PipeOpNOP")` with `FIXME`?